geode-redis/src/main/java/org/apache/geode/redis/internal/executor/RedisKeyInRegion.java

```diff
@@ -46,4 +47,13 @@ public boolean del(ByteArrayWrapper key) {
   public boolean exists(ByteArrayWrapper key) {
     return localRegion.containsKey(key);
   }
+
+  public boolean rename(ByteArrayWrapper oldKey, ByteArrayWrapper newKey) {
+
+    Object value = localRegion.get(oldKey);
```

Comment:
What if get returns null? I think in that case rename should immediately return false.